tsdb/query_executor.go

```diff
@@ -193,6 +198,13 @@ func (q *QueryExecutor) ExecuteQuery(query *influxql.Query, database string, chu
 			case *influxql.ShowStatsStatement, *influxql.ShowDiagnosticsStatement:
 				// Send monitor-related queries to the monitor service.
 				res = q.MonitorStatementExecutor.ExecuteStatement(stmt)
+			case *influxql.CreateSubscriptionStatement, *influxql.DropSubscriptionStatement:
```

Comment:
I need to trigger and event on the Subscriber server whenever a subscription is added or removed.

But I am realizing this is not the right place to tell the Subscriber to update, as only one server in a cluster will actually execute the query. Should I add a Subscriber to the MetaStore instead and have it perform this update after a save? Not entirely sure how to do this part. @pauldix